great_expectations/data_context/data_context/abstract_data_context.py

```diff
@@ -217,6 +217,7 @@ def save_expectation_suite(
         expectation_suite: ExpectationSuite,
         expectation_suite_name: Optional[str] = None,
         overwrite_existing: bool = True,
+        include_rendered_content: bool = False,
```

Comment:
Are we okay with expanding the public signature here? Just want to weigh the pros/cons since it introduces another possible breaking point.

Reply:
Per internal conversations, we should leave this as-is for now.